projects/ccd-case-ui-toolkit/src/lib/shared/components/palette/palette.module.ts

```diff
@@ -119,21 +119,17 @@ import { PaletteService } from './palette.service';
 import { CasePaymentHistoryViewerFieldComponent } from './payment';
 import { ReadPhoneUKFieldComponent, WritePhoneUKFieldComponent } from './phone-uk';
 import {
+  QueryCaseDetailsHeaderComponent,
+  QueryCreateComponent,
+  QueryDetailsComponent,
+  QueryDetailsTableComponent,
+  QueryListComponent,
+  QueryWriteAddDocumentsComponent, QueryWriteDateInputComponent,
```

Comment:
@MateiHMCTS - any reason not to put this on a separate line like the others?

Reply:
IDE added it I guess; Made the change